### ISSUE TITLE
Add option to avoid surface rails when routing

### ISSUE THREAD

**Issue by smaldragon** [auracc/aura-toml#4]
Travel on surface rails is not always ideal, so providing riders a checkbox option to avoid routing through them could be desirable.

A surface field already exists in the AURA data for this purpose.

**Comment by Metriximor** [comment]
What about mixed surface underground rails?

**Comment by smaldragon** [comment]
Yeah, mixed rails would make things more awkward, I feel there would be 2 options to deal with them:

1 - (easiest) Make a general assumption of "is this rail mostly surface or underground" and define them accordingly
2 - (trickier) Make height a non-binary field of `"underground"`, `"mixed"` and `"surface"`. Route along lines according to the height of the points/stations we pass through. (we can't use a binary true/false variable here as a result of places like csa station which is located underground but also hooked up to a surface section of gor)

**Comment by Metriximor** [comment]
Yeah I think option 1 is the best, 2 is too complicated
